structure_tests/licenses_test.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"testing"
+)
+
+var (
+	// Whitelist is the list of packages that we want to automatically pass this
+	// check even if it would normally fail for one reason or another.
+	whitelist = []string{}
+
+	// Blacklist is the set of words that, if contained in a license file, should cause a failure.
+	// This will most likely just be names of unsupported licenses.
+	blacklist = []string{"AGPL", "WTFPL", "AFFERO GENERAL PUBLIC LICENSE", "DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE"}
+)
+
+func TestImageLicenses(t *testing.T) {
+	root := "/usr/share/doc"
+	packages, err := ioutil.ReadDir("/usr/share/doc")
+	if err != nil {
+		t.Fatalf("%s", err)
+	}
+	for _, p := range packages {
```

Comment:
Could you add some t.Logf statements to list the packages we found, and the ones that have valid licenses?